docs/sync.md

```diff
@@ -2,7 +2,13 @@
 
 All course content in PrairieLearn lives in a [Git](https://git-scm.com/) repository, typically hosted on [GitHub](https://github.com/). That repository is synced to/from PrairieLearn.
 
-![High level system structure](high-level.png)
+```d2
+--8<-- "docs/diagrams/high-level.d2"
+```
+
+/// caption
+High level overview of syncing content between PrairieLearn and GitHub.
+///
```

Comment:
What's the story with alt-text or aria-labels for the d2 diagrams? Does this caption get picked up automatically in that role? How about for diagrams without a caption?

Reply:
It looks like diagrams can't support alt text currently, this is a limitation in the mkdocs-d2 plugin.

Reply:
Ok, this isn't a regression, at least. We'll need to worry about this at some point, however.